src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,30 @@
-export const Tabs = () => {};
+/* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
```

Comment:
Disabling ESLint rules related to accessibility (`jsx-a11y/no-noninteractive-element-interactions` and `jsx-a11y/click-events-have-key-events`) should be done with caution. Consider ensuring that keyboard users can also interact with the tabs, for example, by adding keyboard event handlers or using more semantically appropriate elements.